js/components/discussions/thread.vue

```diff
@@ -136,7 +136,7 @@ export default {
             this.detailed = true;
             if (index) {
                 this.$nextTick(() => {
-                    this.$scrollTo(`#${discussionIdAttr}-${index}`);
+                    this.$scrollTo(`#${this.discussionIdAttr}-${index}`);
```

Comment:
A further improvement would be to have an `index` parameter in the `computed.discussionIdAttr` to avoid adding it manually everywhere in the page.